engine/src/flutter/display_list/benchmarking/dl_complexity_metal.cc

```diff
@@ -329,6 +329,12 @@ void DisplayListMetalComplexityCalculator::MetalHelper::drawDiffRoundRect(
   AccumulateComplexity(complexity);
 }
 
+void DisplayListMetalComplexityCalculator::MetalHelper::drawRoundSuperellipse(
+    const DlRoundSuperellipse& rse) {
+  // Round superellipse is only supported by Impeller.
```

Comment:
But for complexity we should probably compute the contribution of the associated approximated rrect instead of just ignoring it.

Reply:
In an earlier version I left a TODO here but then Jonah told me to _not_ implement it, since this class is only used by Skia, which doesn't support RSEs at all and this method will never be called.

(In a future version we might allow Skia to support RSE via Bezier approximation, but even then the algorithm should be derived from experiments AFAIK.)

Reply:
We only compute complexity values when we are using the Skia backend, true. But, we've implemented the drawing and clipping methods for the Skia backend by forwarding them to the rrect equivalents - thus, Skia *DOES* render RSEs, it just renders an approximation for them. This method (and its clipRSE counterpart) should reflect that by adding in the complexity of the associated RRect.

Please do implement this by forwarding to the RRect method with the approximated RRect.

It's not that Skia is a high priority, but this is a one-line obvious implementation.

```
  return drawRoundRect(rse.approximation());
```

Reply:
Thanks for explaning, now I understand.

Done. https://github.com/flutter/flutter/pull/160883/files#diff-3279d9eadeb9e2e9e827dcff28c305723f2fe75238da9da75d4cf8987d6f53daR335